node_launcher/node_set/node_set.py

```diff
@@ -32,6 +34,7 @@ def __init__(self, network: Network):
             bitcoin=self.bitcoin
         )
         self.lnd_client = LndClient(self.lnd)
+        self.setup_tor = setup_tor()
```

Comment:
Pass in the NodeSet here so you have access to the configuration files

```suggestion
        self.setup_tor = SetupTor(self)
```